htdocs/modulebuilder/template/core/triggers/interface_99_modMyModule_MyModuleTriggers.class.php

```diff
@@ -213,6 +213,14 @@ public function runTrigger($action, $object, User $user, Translate $langs, Conf
 			//case 'LINEBILL_UPDATE':
 			//case 'LINEBILL_DELETE':
 
+			// Recurring Bills
+			//case 'BILLREC_CREATE':
+			//case 'BILLREC_MODIFY':
+			//case 'BILLREC_DELETE':
+			//case 'BILLREC_AUTOCREATEBILL':
```

Comment:
This one has been removed recently. Can you remove it ?

Reply:
done